chat-widget/src/components/livechatwidget/common/startChat.ts

```diff
@@ -81,6 +81,15 @@ const initStartChat = async (chatSDK: any, chatConfig: ChatConfig | undefined, g
     try {
         let isStartChatSuccessful = false;
```

Comment:
I understand we got little lazy, but let's move the initialize line below authentication :)

Reply:
done